src/hints/math/divMod.test.ts

```diff
@@ -88,4 +88,21 @@ describe('DivMod', () => {
       );
     }
   );
+
+  test('should throw an error when rhs is zero', () => {
+    const hint = divModParser.parse(DIV_MOD_HINT);
+    const vm = new VirtualMachine();
+    vm.memory.addSegment();
+    vm.memory.addSegment();
+
+    const lhsValue = new Felt(17n);
+    const rhsValue = new Felt(0n);
+
+    vm.memory.assertEq(vm.ap, lhsValue);
+    vm.memory.assertEq(vm.ap.add(1), rhsValue);
+
+    expect(() => {
+      divMod(vm, hint.lhs, hint.rhs, hint.quotient, hint.remainder);
+    });
```

Comment:
You're expecting the  `divMod` function to throw the error `CannotDivideByZero()`, it is not reflected in the test

It should be of the form `expect(() => foo()).toThrow(new MyError())`